src/highdicom/sr/sop.py

```diff
@@ -180,67 +180,168 @@ def __init__(
+        ----------
+        collection: Dict[Tuple[str, str], List[pydicom.dataset.Dataset]]
+            Items of the Referenced SOP Sequence grouped by Study and Series
+            Instance UID
+
+        Returns
+        -------
+        List[pydicom.dataset.Dataset]
+            Items containing the Study Instance UID and the
+            Referenced Series Sequence attributes
+
+        """
+        study_collection: Dict[str, List[Dataset]] = defaultdict(list)
+        for (study_uid, series_uid), instance_items in collection.items():
+            series_item = Dataset()
+            series_item.SeriesInstanceUID = series_uid
+            series_item.ReferencedSOPSequence = instance_items
+            study_collection[study_uid].append(series_item)
+
+        ref_items = []
+        for study_uid, series_items in study_collection.items():
+            study_item = Dataset()
+            study_item.StudyInstanceUID = study_uid
+            study_item.ReferencedSeriesSequence = series_items
+            ref_items.append(study_item)
+
+        return ref_items
+
+    def _collect_predecessors(
+        self,
+        previous_versions: List[Dataset]
+    ) -> List[Dataset]:
+        """Collect predecessors of the SR document.
+
+        Parameters
+        ----------
+        previous_versions: List[pydicom.dataset.Dataset]
+            Metadata of instances that represent previous versions of the
+            SR document content
+
+        Returns
+        -------
+        List[pydicom.dataset.Dataset]
+            Items of the Predecessor Documents Sequence
+
+        """
+        collection: Dict[str, List[Dataset]] = defaultdict(list)
+        for pre in previous_versions:
+            pre_instance_item = Dataset()
+            pre_instance_item.ReferencedSOPClassUID = pre.SOPClassUID
+            pre_instance_item.ReferencedSOPInstanceUID = pre.SOPInstanceUID
+            key = (pre.StudyInstanceUID, pre.SeriesInstanceUID)
+            collection[key].append(pre_instance_item)
+        return self._create_references(collection)
+
+    def _collect_evidence(
+        self,
+        evidence: List[Dataset],
+        content: Dataset
+    ) -> Tuple[List[Dataset], List[Dataset]]:
+        """Collect evidence for the SR document.
+
+        Any `evidence` that is referenced in `content` via IMAGE or
+        COMPOSITE content items will be grouped together for inclusion in the
+        Current Requested Procedure Evidence Sequence and all remaining
+        evidence will be grouped for potential inclusion in the Pertinent Other
+        Evidence Sequence.
+
+        Parameters
+        ----------
+        evidence: List[pydicom.dataset.Dataset]
+            Metadata of instances that serve as evidence for the SR document
+            content
+        content: pydicom.dataset.Dataset
+            SR document content
+
+        Returns
+        -------
+        Tuple[List[pydicom.dataset.Dataset], List[pydicom.dataset.Dataset]]
```

Comment:
As discussed previously, my preferred style is to list elements of the output tuple as individual named outputs as this leads to clearer docs in my opinion. Take it or leave it.

Reply:
I agree and I think we should adopt this style of documenting the return value. However, I would like the same style to be consistently applied throughout the library and suggest updating docstrings via a separate PR.

Reply:
I did some grepping tosee whether there are in fact any other instances of this in the codebase, and I found a couple. Dumping here for reference:
- `highdicom.seg.content.DimensionIndexSequence.get_index_values`
- `highdicom.spatial.map_pixel_into_coordinate_system` and `highdicom.spatial.map_coordinate_into_pixel_matrix`. These may actually be better left as they are as I imagine the output will typically not be destructered